docs/source/logging/awscloud.md

```diff
@@ -0,0 +1,93 @@
+# Implementing the Logging System
```

Comment:
this section structure is a bit confusing: 
<img width="259" alt="Screenshot 2024-06-10 at 5 50 48 PM" src="https://github.com/OSOceanAcoustics/echodataflow/assets/11621647/d60416a3-5867-41c3-a05c-0ad35db66b0e">
It is better to be 
Logging
 * dask workers
 * aws cloudwatch
 * kafka ekl
 
So better to split amazon watchtower and dask workers.